docker/Dockerfile-workers

```diff
@@ -23,23 +23,16 @@ RUN dnf -y install \
     libseccomp-devel \
     ostree-devel \
     systemd-devel \
+    && dnf update -y \
     && dnf clean all
 
 ENV OPM_SRC="/src"
-ENV OPM_TAG="v1.17.0"
+ENV OPM_TAG="v1.17.3"
```

Comment:
Should we go with v1.18.0?

Reply:
I will remove building the OPM and add binary here. Thanks @lipoja

Reply:
changed